### PR TITLE
Fix Application Registry app update retry

### PR DESCRIPTION
**Pull request by franpog859**
**Description**

Changes proposed in this pull request:

- Fix Application Registry app update retry

**Related issue(s)**

See the issue [#8019](https://github.com/kyma-project/kyma/issues/8019)

### CODE REVIEW

File: components/application-registry/internal/metadata/applications/repository.go [hunk 1]
```diff
@@ -21,12 +21,12 @@ const (
 
 // Manager contains operations for managing Application CRD
 type Manager interface {
-	Update(application *v1alpha1.Application) (*v1alpha1.Application, error)
+	Update(appName *v1alpha1.Application) (*v1alpha1.Application, error)
```
Comment (Szymongib) [inline components/application-registry/internal/metadata/applications/repository.go:24]:
```suggestion
	Update(application *v1alpha1.Application) (*v1alpha1.Application, error)
```

Comment (franpog859) [reply components/application-registry/internal/metadata/applications/repository.go:24]: True ^^

**Pull request merged**